skimage/exposure/exposure.py

```diff
@@ -463,3 +464,48 @@ def adjust_sigmoid(image, cutoff=0.5, gain=10, inv=False):
 
     out = (1 / (1 + np.exp(gain * (cutoff - image / scale)))) * scale
     return dtype(out)
+
+
+def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
+                    upper_percentile=99, method='linear'):
+    """Detemine if an image is low contrast.
+
+    Parameters
+    ----------
+    image : array-like
+        The image under test.
+    fraction_threshold : float, optional
+        The low contrast fraction threshold.
+    lower_bound : float, optional
+        Disregard values below this percentile when computing image contrast.
+    upper_bound : float, optional
+        Disregard values above this percentile when computing image contrast.
+    method : str, optional
+        The contrast determination method.  Right now the only available
+        option is "linear".
+
+    Returns
+    -------
+    out : bool
+        True when the image is determined to be low contrast.
+
+    Examples
+    --------
+    >>> image = np.linspace(0, 0.04, 100)
+    >>> is_low_contrast(image)
+    True
+    >>> image[-1] = 1
+    >>> is_low_contrast(image)
+    True
+    >>> is_low_contrast(image, upper_percentile=100)
+    False
+    """
+    image = np.asanyarray(image)
+    if image.ndim >= 2:
+        image = rgb2gray(image)
```

Comment:
No, I think this is terrible! Before, it worked fine on n-d images, now it will break horrendously.

Maybe `if image.ndim == 3 and image.shape[2] == 3:`?